plugins/defaultplugins/common/model/l3/l3.proto

```diff
@@ -33,26 +33,34 @@ message ArpTable {
 
 /* Proxy ARP ranges */
 message ProxyArpRanges {
-    message ProxyArpRange {
-        string range_ip_start = 1;                  /* low (starting) IP address of the proxy ARP range */
-        string range_ip_end = 2;                    /* high (last) IP address of the proxy ARP range  */
+    message RangeList {
+        string name = 1;                                /* unique proxy ARP range config identifier */
+        message Ranges {
+            string first_ip = 1;                        /* first (starting) IP address of the proxy ARP range */
+            string last_ip = 2;                         /* last (ending) IP address of the proxy ARP range  */
+        }
+        repeated Ranges ranges = 2;
     }
-    repeated ProxyArpRange proxy_arp_ranges = 100;  /* list of proxy arp ranges */
+    repeated RangeList ranges = 100;                    /* list of proxy arp ranges */
```

Comment:
This might be better named with `range_list` so it's not confused with the inner field `ranges`.